src/HasNotes.php

```diff
@@ -12,9 +12,12 @@ public function notes(): MorphMany
         return $this->morphMany($this->getNoteModelClassName(), 'model', 'model_type', $this->getModelKeyColumnName())->latest('id');
     }
 
-    public function setNote(string $note): Note
```

Comment:
I think we should keep this API the same. Or make the `group` the second argument